pdf.go

```diff
@@ -17,6 +115,11 @@ func ConvertPDF(r io.Reader) (string, map[string]string, error) {
 	}
 	defer f.Done()
 
+	// Verify if pdf has images or is pdf only-text
+	if PDFHasImage(f.Name()) {
```

Comment:
Ok, so you need to move this so that it's only enabled when OCR is enabled (otherwise PDFs which have images and text will be ignored if OCR hasn't been built in).